assets/css/main.css

```
@@ -91,6 +91,33 @@ a:hover {
 
 /* FAQ
 ================================ */
+.accordion-button::after {
+  background-image: url(../img/plus-drop-down.svg);
+  }
+
+.accordion-button:not(.collapsed)::after {
+  background-image: url(../img/minus-drop-down.svg);
+}
+
+.accordion-button:focus {
+  border-color: rgba(0,0,0,.125);
+  box-shadow: none;
+}
+
+.accordion-button:not(.collapsed) {
+  color: black;
```

It's usually better practice to use CSS Variables instead of color literals for values that get reused often. In this case, if we ever wanted to change the text color from `black` to a slightly different shade, we just swap it inside the CSS variable. The list of variables is defined in the `:root` selector at the top of this CSS file.
```suggestion
  color: var(--color-dark-font);
```